app/services/rates_importer_service.rb

```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 class RatesImporterService
+  def clear!(rating_year)
+    puts rating_year
+    RatesBill.joins(:property).where("properties.rating_year": rating_year).delete_all
+    RatesPayer.joins(:property).where("properties.rating_year": rating_year).delete_all
+    Property.where(rating_year: rating_year).delete_all
```

Comment:
@Br3nda is this going to delete entries in prod? or should we manually take care of those 2019 entries?

Reply:
Yes, it's going to the *2019* records, on production. like i just did on staging.

Reply:
It's going to delete  RatesBill and RatesPayer and Property records, for 2019